benefits/core/templates/core/base.html

```diff
@@ -111,6 +111,9 @@
           <li>
             <a class="footer-link m-0 p-0" href="https://cdt.ca.gov/privacy-policy/" target="_blank" rel="noopener noreferrer">{% translate "Privacy Policy" %}</a>
           </li>
+          <li class="ms-auto">
```

Comment:
`ms-auto` means "margin _start_: auto", and "start" means `left` if the page is set to a language that reads from left to right.